src/components/Table/Table.js

```diff
@@ -1,75 +1,63 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-
 // @CHILD CLASSES
 import TableHeader from './TableHeader';
 import TableBody from './TableBody';
-
 class Table extends Component {
-
     constructor() {
         super();
-
         this.state = {
             activeRowID: [],
-            myUserList: []
+            myUserList: [],
+            searchValue: 0
         }
-
     }
-
-
     componentWillReceiveProps(nextProps) {
         this.setState({
             myUserList: nextProps.data
         });
     }
-
-    // (1) TODO HOME WORK
     deleteFromTable = () => {
         const array = this.state.myUserList;
         const activeRows = this.state.activeRowID;
-
         for (let i = 0; i < array.length; i++) {
             if (activeRows.includes(array[i])) {
                 delete array[i];
             }
-
         };
-
         this.setState({
             myUserList: array
         });
-
-
     }
-
+    searchFromTable = (e) =>{
+            const {value} = e.target
+        this.setState({
+            searchValue: value })
+    }
     activeRowHanlder = (value) => {
         const array = this.state.activeRowID;
         if (array.indexOf(value) > -1) {
             let indexToRemove = array.indexOf(value);
             delete array[indexToRemove];
-
         } else {
             array.push(value);
-
         }
-
         localStorage.setItem('activeUsers', array);
         this.setState({
             activeRowID: array
         });
-
-
     }
-
     render() {
-        console.log(this.state.activeRowID)
         const { data } = this.props;
         const headerColumnNames = this.state.myUserList.length > 0 ? Object.keys(this.state.myUserList[0]) : [];
         headerColumnNames.unshift('checkbox');
         const activeColumns = ['checkbox', 'id', 'title'];
         return (
             <div>
+                <input
```

Comment:
input search should be another class
example: in future u need use same search component in another case.